### PR TITLE
Broader synopsis in general report, much better :D

### PR DESCRIPTION
**Pull request by northwestwitch**
fir #3210 

From this:

image

To this:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 1]
```diff
@@ -161,26 +161,28 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
...
-	              {% for pheno in case.phenotype_terms %}
-	                <li>
-	                {{pheno.feature}} - (<a href="{{pheno.hpo_link}}" target="_blank">{{pheno.phenotype_id}})</a>
-	                </li>
-	              {% endfor %}
-	            </ul>
-	          {% else %}
-	            No phenotype terms associated.
-	          {% endif %}
+	        <td class="ml-3 d-flex align-items-start flex-column">
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:173]: 👍

**Review by dnil — APPROVED** [review]
You win! 😆

**Comment by northwestwitch** [comment]
@dnil forgot to claim the issue and didn't notice I did, so we ended up working on the same one.

File: scout/server/blueprints/cases/templates/cases/case_report.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case_report.html:173]: Do I win for this layout?

Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/case_report.html:173]: Yes, more flex in there!

Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/case_report.html:173]: To my defence, I was first, and it looked about the same. 😉

**Pull request merged**